saas_public/doc/index.rst

```diff
@@ -4,7 +4,10 @@
 
 Installation
 ============
-* Installation is similar to SaaS Base module
+
+* Follow instruction of `Job Queue <https://github.com/OCA/queue/tree/12.0/queue_job>`__ module.
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
+* Restart the odoo as required by the Job Queue module
```

Comment:
You shall not copy-paste instruction. Just use proper reference.  ``* Installation is similar to SaaS Base module`` doesn't look friendly (no link to the module). Use link to the github